week3/aniketsaoji/anagrams.java

```diff
@@ -0,0 +1,44 @@
+package anagrams;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class anagrams {
+	
+	public static ArrayList<Character> returnList(String word) {
```

Comment:
What about renaming `returnList` to something more meaningful. Also do you need to use an `ArrayList<Character>` here?